scripts/perf-compare.sh

```diff
@@ -6,6 +6,9 @@ set -eo pipefail
 SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
 cd "$SCRIPT_DIR" || exit 1
 
+SHAPER=$(mktemp)
+sed -e '1,/```mdtest-input shaper.zed/d' ../docs/integrations/zeek/shaping-zeek-ndjson.md | sed -e '/```/,$d' > "$SHAPER"
```

Comment:
Nits: All-caps is for environment variables and shouting. And just invoke sed once.
```suggestion
shaper=$(mktemp)
sed -e '1,/```mdtest-input shaper.zed/d' -e '/```/,$d' ../docs/integrations/zeek/shaping-zeek-ndjson.md > "$shaper"
```